source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -103,14 +110,17 @@ void SwiftLanguageRuntime::SetupReflection() {
 
   auto module_list = GetTargetRef().GetImages();
   module_list.ForEach([&](const ModuleSP &module_sp) -> bool {
-    std::string module_path = module_sp->GetFileSpec().GetPath();
     auto *obj_file = module_sp->GetObjectFile();
+    if (!obj_file)
+        return false;
     Address start_address = obj_file->GetHeaderAddress();
     auto load_ptr = static_cast<uintptr_t>(
         start_address.GetLoadAddress(&(m_process->GetTarget())));
     if (load_ptr == 0 || load_ptr == LLDB_INVALID_ADDRESS)
       return false;
-    reflection_ctx->addImage(swift::remote::RemoteAddress(load_ptr));
+    std::string module_path = module_sp->GetFileSpec().GetPath();
+    if (module_path.find("swift") != std::string::npos)
```

Comment:
why does this work?

Reply:
That was just an hack to see if it worked on the bots. I implemented the correct logic now.